draft-pda-protocol.md

```diff
@@ -375,23 +378,24 @@ enum { prio(0), hits(1) } PAProto;
 opaque Url<1..2^16-1>;
 ~~~
 
-The first field is the PA task. The `helper_urls` defines the helpers' endpoint
-URLs. The `collector_config` is the HPKE configuration of the collector
-(described in {{hpke-config}}). The `batch_size` field encodes the *batch size*,
-the number of input shares accumulated by each aggregator before emitting its
-output share. The `proto` field identifies the specific PA protocol in use. The
-rest of the structure contains any protocol-specific parameters that are
-required.
+* `task`: The PA task.
+* `helper_urls`: The helpers' endpoint URLs.
+* `collector_config`: The HPKE configuration of the collector (described in
+  {{hpke-config}}). [OPEN ISSUE: Maybe the collector's HPKE config should be
+  carried by the collect request?]
```

Comment:
This would allow the leader to impersonate the collector when it relays the HPKE config to the helper -- but as you say, it's an open issue so we don't have to solve it right now.

Reply:
Actually, at the moment the helper is "pre-configured" with the PAParam (and thus the collectors HPKE config). But yeah, this something we have to work out.